var/spack/repos/builtin/packages/libflame/package.py

```diff
@@ -119,3 +106,23 @@ def darwin_fix(self):
         # The shared library is not installed correctly on Darwin; fix this
         if self.spec.satisfies('platform=darwin'):
             fix_darwin_install_name(self.prefix.lib)
+
+
+class Libflame(LibflameBase):
+    """libflame is a portable library for dense matrix computations,
+    providing much of the functionality present in LAPACK, developed
+    by current and former members of the Science of High-Performance
+    Computing (SHPC) group in the Institute for Computational
+    Engineering and Sciences at The University of Texas at Austin.
+    libflame includes a compatibility layer, lapack2flame, which
+    includes a complete LAPACK implementation."""
+
+    homepage = "https://www.cs.utexas.edu/~flame/web/libFLAME.html"
+    url = "https://github.com/flame/libflame/archive/5.1.0.tar.gz"
+    git = "https://github.com/flame/libflame.git"
+
+    version('master', branch='master')
+    version('5.2.0', sha256='997c860f351a5c7aaed8deec00f502167599288fd0559c92d5bfd77d0b4d475c')
+    version('5.1.0', sha256='e7189b750890bd781fe773f366b374518dd1d89a6513d3d6261bf549826384d1')
+
+    provides('flame@5.2', when='@5.2.0')
```

Comment:
What about when at 5.1.0? I assume it still provides flame.

Reply:
Added provides entry for 5.1.0 version